core/internal/cltest/cltest.go

```diff
@@ -268,11 +269,11 @@ func (ta *TestApplication) StartAndConnect() error {
 		return err
 	}
 
-	return ta.WaitForConnection()
+	return ta.waitForConnection()
 }
 
-// WaitForConnection wait for the StartAndConnect callback to be called
-func (ta *TestApplication) WaitForConnection() error {
+// waitForConnection wait for the StartAndConnect callback to be called
+func (ta *TestApplication) waitForConnection() error {
```

Comment:
Why was this made private?

Reply:
Because it is no longer used outside of this module, nor should it be used.

Reply:
Maybe add the reasoning in the PR comment or commit message?  As it looks outside of the context of the PR.  It might speed up the back and forth.